google-cloud-dlp/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=DLP API
+--exclude lib/google/cloud/dlp/v2beta1
```

Comment:
@quartzmo That sounds right to me.

@landrito Could you confirm that this line is intended to exclude the protoc output? If so, I can update [the template](https://github.com/googleapis/toolkit/blob/2c2323e5dea1c51cb3249b045719f6e8175e4a92/src/main/resources/com/google/api/codegen/ruby/yardopts.snip#L4), which seems to use the wrong value here.